src/index.js

```diff
@@ -74,19 +86,96 @@ function doThis() {
   $('.total_rooms').text(hotel.totalRoomsAvailableToday(currentDate));
   $('.total_rev').text(hotel.totalRevenueToday(currentDate));
   $('.total_perc').text(hotel.percentRoomsOccupied(currentDate));
-  console.log(hotel.users)
 }
 
 function orThis() {
   $('.login_form').toggle('hidden');
   $('header').toggle('hidden');
   $('.customer_dashboard').toggle('hidden');
-  let id = parseInt($('.login_username').val())
-  hotel.findUser(id)
-  user = new User(hotel.currentUser, hotel.bookings, hotel.rooms)
-  console.log(user)
-  $('.header_p_span').text(user.name.split(' ')[0])
-  $('.customer_total').text(user.totalSpent())
+  let id = parseInt($('.login_username').val());
+  hotel.findUser(id);
+  user = new User(hotel.currentUser, hotel.bookings, hotel.rooms);
+  $('.header_p_span').text(user.name.split(' ')[0]);
+  $('.customer_total').text(user.totalSpent());
+  displayBookings()
 }
 
+function displayBookings() {
+  let allBookings = user.allUserBookings()  
+  allBookings.forEach(currentBooking => {
+      $('.booking_body').append(`
+      <tr class="room-number" data-room="${currentBooking.roomNumber}">
+      <td>${currentBooking.roomNumber}</td>
+      <td>${currentBooking.date}</td>
+      </tr>
+      `)
+    })
+  }
+
+function sortByType(table, prop) {
+  let rooms = table
+  if(prop === 'roomType') {
+    return rooms.sort((a, b) => {
+      if(a[prop] < b[prop]) { 
+        return -1; 
+      }
+      if(a[prop] > b[prop]) { 
+        return 1 
+      }
+    })
+  }
+  return rooms.sort((a, b) => a[prop] - b[prop]);
+}
+
+function displayRoomsAvailable(table) {
+  $('.room_body').empty();
+  if(table) {
+    table.forEach(room => {
+      $('.room_body').append(`
+      <tr class="table_room-number" data-room="${room.number}">
+        <td>${room.number}</td>
+        <td>${room.roomType}</td>
+        <td>${room.bidet}</td>
+        <td>${room.bedSize}</td>
+        <td>${room.numBeds}</td>
+        <td>$${room.costPerNight}</td>
+      </tr>
+      `)
+    })
+  } else {
+  let today = $('.datepicker').val()
+  let todaysRooms = user.roomsAvailableToday(today)
```

Comment:
Turing naming conventions for functions suggest using a verb in the function name (like showRoomsAvailableToday). Doesn't actually really matter, though. I'm really just struggling to find anything actionable to offer you. Basically, just disregard this comment. Code looks good and reviewing your approach to the fetch and HTML tables has been helpful for me. Thanks, man